src/NuGetGallery.Core/GitHub/IGitHubUsageConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NuGetGallery.GitHub
+{
+    public interface IGitHubUsageConfiguration
+    {
+        NuGetPackageGitHubInformation GetPackageInformation(string packageId);
```

Comment:
We're not super great at commenting interfaces in the Gallery code base, but we really should. Could you add comments here? Mention that `packageId` is required. Mention that `NuGetPackageGitHubInformation` is valid even if the `packageId` does not exist.